homeassistant/components/homematicip_cloud/services.py

```diff
@@ -217,15 +217,6 @@ async def async_call_hmipc_service(service: ServiceCall) -> None:
     )
 
 
-async def async_unload_services(hass: HomeAssistant):
```

Comment:
I think it would be better to split this PR between `isy994` and `homematicip_cloud`.

In `homematicip_cloud`, the tests need adjusting, and all the methods would benefit from removing `hass` argument and using `service.hass` property instead.